haproxy/dockerdir/usr/local/bin/check_pxc.sh

```diff
@@ -17,6 +17,7 @@ if [ -f "$PATH_TO_SECRET/OK_IF_DONOR" ]; then
     OK_IF_DONOR=$(/bin/cat $PATH_TO_SECRET/OK_IF_DONOR)
 fi
 
+VERBOSE=${VERBOSE:-1}
```

Comment:
@2ZZ this script does not have access to enc var. So, you need to get it from file

```suggestion
if [ -f "$PATH_TO_SECRET/VERBOSE" ]; then
    VERBOSE=$(/bin/cat $PATH_TO_SECRET/VERBOSE)
fi

VERBOSE=${VERBOSE:-1}
```